packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -527,22 +527,6 @@ class FlutterPlugin implements Plugin<Project> {
         if (project.hasProperty('track-widget-creation')) {
             trackWidgetCreationValue = project.property('track-widget-creation').toBoolean()
         }
-        String compilationTraceFilePathValue = null
-        if (project.hasProperty('compilation-trace-file')) {
-            compilationTraceFilePathValue = project.property('compilation-trace-file')
-        }
-        Boolean createPatchValue = false
-        if (project.hasProperty('patch')) {
-            createPatchValue = project.property('patch').toBoolean()
-        }
-        Integer buildNumberValue = null
```

Comment:
This didn't seem hot code related, but it was only provided if we were creating a patch